<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1427</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LCD Constant Search</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lbImage">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>20</y>
      <width>711</width>
      <height>441</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pbStartSearch">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbCancleSearch">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel Search</string>
    </property>
   </widget>
   <widget class="QWidget" name="myWidget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>631</width>
      <height>351</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="constPlot" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>631</width>
      <height>351</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1427</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
